### PR TITLE
[ADD] finance/fiscal_localization: Austria (i.e. SAF-T export)

### PR DESCRIPTION
**Pull request by svfu-odoo**
This PR adds the (currently missing) documentation for the Austrian fiscal localization.

It i.e. includes information about the new SAF-T export for Austria.

Community PR: https://github.com/odoo/odoo/pull/135485
Enterprise PR: https://github.com/odoo/enterprise/pull/47405
Documentation PR: https://github.com/odoo/documentation/pull/5859

task-3354661

### CODE REVIEW

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:17]: not necessary

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:21]: not necessary

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:25]: not necessary

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:29]: not necessary

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:46]: The sentence is not so clear, not sure the good meaning is conveyed. Try maybe to put it in active form starting with "The mapping from your company's chart of accounts.."

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:47]: add this in the see also box

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:61]: The following explains how to ensure that all the information needed for the SAF-T export is available. If missing information is detected during the export process, a warning message will appear that lists the missing information.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:54]: The Austrian tax office may request a SAF-T report on an on-demand basis. The Austrian SAF-T Export module allows export of a SAF-T document (in XML format).

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:67]: Go to guilabel :`Settings --> General Settings`, in the :guilabel:`Companies` section, click
:guilabel:`Update Info` and fill in the following information:

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:71]: Fill in the :guilabel:`Address` and :guilabel:`Phone` fields. Entering (at least)  :guilabel:`Street`, :guilabel:`City`, :guilabel:`ZIP` and :guilabel:`Country`.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:76]: You must have at least one **contact person** linked to your company in the :guilabel:`Contacts` app:

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:95]: at the end of the section

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:111]: the ref link can be directly on "SAF-T"

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:103]: Similarly, any number can be used instead of `1000` to map an account to any account in the SAF-T COA

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:107]: Furthermore, it installs a (suggested) mapping for many of the accounts

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:113]: It includes a button to access a list of these problematic accounts.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:119]: Fill in the :guilabel:`Address` of each partner that
appears in your invoices, vendor bills, or payments through the :guilabel:`Contacts` app.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:132]: 2. Open the drop-down menu on the right side of the :guilabel:`PDF` button.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:126]: Generating the SAF-T

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (chiaraprattico) [inline content/applications/finance/fiscal_localizations/austria.rst:129]: To export the SAF-T document follow the next steps:

**Review by chiaraprattico — CHANGES_REQUESTED** [review]
Hi @svfu-odoo nice one, I left a few comments, let me know if you need any help with them.

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (svfu-odoo) [reply content/applications/finance/fiscal_localizations/austria.rst]: I added a link to the mentioned regulations. They give an overview about how it should look like approximately. Is it okay?

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (svfu-odoo) [reply content/applications/finance/fiscal_localizations/austria.rst]: See above

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (svfu-odoo) [reply content/applications/finance/fiscal_localizations/austria.rst]: I put the link on "export the SAF-T" since it seemed clearer to me. Is it okay?

File: content/applications/finance/fiscal_localizations/austria.rst
Comment (svfu-odoo) [reply content/applications/finance/fiscal_localizations/austria.rst]: I used `allows exporting a SAF-T document` instead of `allows export of a ...` (it did not seem right to me). Is it okay?

**Comment by svfu-odoo** [comment]
Thanks for the feedback

**Review by xpl-odoo — APPROVED** [review]
Hi @svfu-odoo !

I pushed some changes on another commit to go quickly. Normally, everything should be good to go. However, have a look at least at the COA section to make sure I didn't change the meaning of anything by trying to simplify a bit.

Note, you will need to squash the commits before merging.

Don't hesitate to reach out if you have any questions regarding my changes.

@robodoo delegate+

**Comment by svfu-odoo** [comment]
Hi @xpl-odoo 
Thank you. It is just better now.

**Comment by xpl-odoo** [comment]
> Hi @xpl-odoo Thank you. It is just better now.

As you probably have noticed, you can r+ to merge the PR whenever you prefer. I assume you are waiting for the dev PRs to be merged themselves :)

**Comment by svfu-odoo** [comment]
Yes, thanks a lot. Due to conflicting PRs the other PRs may still take some time unfortunately.

**Comment by svfu-odoo** [comment]
Rebased to base match the other PRs

**Comment by jco-odoo** [comment]
@svfu-odoo I don't know if you can give the ok now, but this is the only PR still blocking the merge normally.

**Comment by svfu-odoo** [comment]
@robodoo r+

**Comment by svfu-odoo** [comment]
> @svfu-odoo I don't know if you can give the ok now, but this is the only PR still blocking the merge normally.

Should be good. Thanks

**Pull request closed without merging**
